specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/DatabaseAuditingSettingsList.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "blobauditingtest-6852",
+    "serverName": "blobauditingtest-2080",
+    "databaseName": "testdb",
+    "api-version": "2015-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/blobauditingtest-6852/providers/Microsoft.Sql/servers/blobauditingtest-2080/databases/testdb",
```

Comment:
This id is wrong, it doesn't have the auditingSettings or resource name in the ID